static/scss/answers/templates/vertical-standard.scss

```diff
@@ -21,6 +21,12 @@
     }
   }
 
+  &-resultsMidContainer {
+    flex-direction: column;
+    flex-grow: 1;
+    width: 100%;
```

Comment:
is this width 100% necessary? same question above

Reply:
it's needed for IE11. the flexbox behavior for ie is different compare to other browsers. Without this, the card size will spill over to the right of the page